.github/workflows/python-package.yml

```diff
@@ -16,11 +16,22 @@ jobs:
         strategy:
             fail-fast: true
             matrix:
-                os: [ubuntu-latest]
-                python-version: ['3.10', '3.11']  # Py 3.10.14 as of 2025-04
-                astropy-version: ['<6.1']         # KPNO+NERSC, 2025/04
-                fitsio-version: ['<2']            # latest
-                numpy-version: ['<1.23']          # to keep asscalar, used by astropy
+              include:
+
+                - os: ubuntu-latest
+                  python-version: '3.10'          # Py 3.10.14 as of 2025-04
+                  astropy-version: '<6.1'         # KPNO+NERSC, 2025/04
+                  fitsio-version: '<2'            # latest
+                  numpy-version: '<1.23'          # to keep asscalar, used by astropy
+                  scipy-version: '<1.9'           # for old numpy compatibility
+
+                - os: ubuntu-latest
+                  python-version: '3.12'
+                  astropy-version: '<7.0'         # KPNO+NERSC, 2025/04
+                  fitsio-version: ''              # latest
+                  numpy-version: '<2.3'
+                  scipy-version: ''               # latest
+
         env:
             DESIUTIL_VERSION: 3.5.0
```

Comment:
This is already a slightly older version of desiutil (latest = 3.5.2), but I would suggest simply doing `pip install desiutil` and not picking a version at all, unless that introduces additional problems.